common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -204,9 +206,7 @@ private TardisNavLocation scanUpForPosition(TardisNavLocation startingLocation)
         while (currentPos.getY() < level.getMaxBuildHeight()) {
             if (!level.getBlockState(currentPos).getMaterial().isSolidBlocking()) {
                 // Check if the Shell can be physically in the location.
-                if (level.getBlockState(currentPos.below()).getMaterial().isSolidBlocking()
-                        && !level.getBlockState(currentPos.above()).getMaterial().isSolidBlocking()
-                        && !level.getBlockState(currentPos.below()).is(Blocks.WATER)) {
+                if (level.getBlockState(currentPos.below()).getMaterial().isSolidBlocking() && !level.getBlockState(currentPos.above()).getMaterial().isSolidBlocking() && !level.getBlockState(currentPos.below()).is(Blocks.WATER)) {
```

Comment:
The check for the Water Block will mean it will ignore if its Fluid Type is source or flowing. A possible side effect is that waterlogged blocks which has their water spread out can be used to prevent landing. Do we want to check for fluid state instead so that we can do more granular checks?